modules/transposition.nix

```
@@ -67,7 +67,11 @@ in
     perInput =
       system: flake:
       mapAttrs
-        (attrName: attrConfig: flake.${attrName}.${system})
+        (attrName: attrConfig:
+          flake.${attrName}.${system} or (throw ''
+            Attemt to access non existent attribute ${attrName}.${system} of flake ${flake}.
+          '')
```

Often this will be recoverable. For instance for `packages`, we can return `{ }`, so that `inputs'.foo.packages?bar` works.

```suggestion
          flake.${attrName}.${system} or (
            if perInputDefault?${attrName}
            then perInputDefault.${attrName} system
            else throw ''
                Attemt to access non-existent attribute ${lib.escapeNixIdentifier attrName}.${lib.escapeNixIdentifier system} of flake input ${lib.escapeNixIdentifier flake}.
              '')
```